redbot/core/bot.py

```diff
@@ -120,14 +120,17 @@ def __init__(self, cli_flags, bot_dir: Path=Path.cwd(), **kwargs):
 
     async def is_admin(self, member: discord.Member):
         """Checks if a member is an admin of their guild."""
-        admin_role = await self.db.guild(member.guild).admin_role()
-        return any(role.id == admin_role for role in member.roles)
+        admin_roles = await self.db.guild(member.guild).admin_role()
+        return (not admin_role or
```

Comment:
First of all, there's a typo here

Second of all, this leads to unexpected behaviour; we can't assume every member of the server is an admin simply because the role isn't set. (I know this because I made this exact mistake when I implemented this function, only for it to be fixed in #1166)